### PR TITLE
bug: expanding model when query param showExtensions=true exists

### PR DESCRIPTION
**Pull request by tim-lai**
### Description

In the ObjectModel, insert a table cell `td` into the table row `tr`. Then insert empty space `&nbsp;` into the table cell.

### Motivation and Context

1. load SwaggerUI in dev mode
2. use 2 query parameters, e.g. http://localhost:3200/?docExpansion=full&showExtensions=true
3. expand any model
4. console error
```
warning.js:33 Warning: validateDOMNesting(...): Whitespace text nodes cannot appear as a child of <tr>.
```

### How Has This Been Tested?

Following the above steps no longer produces the error with this fix

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by tim-lai** [comment]
please build

File: src/core/components/model.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import PropTypes from "prop-types"
 
 export default class Model extends ImmutablePureComponent {
   static propTypes = {
-    schema: ImPropTypes.orderedMap.isRequired,
+    schema: ImPropTypes.map.isRequired,
```
Comment (shockey) [inline src/core/components/model.jsx]: likewise, looks like #5914 snuck in here!
